docs/content/Schema/Reference/pre-aggregations.md

```diff
@@ -493,6 +493,30 @@ cube(`Orders`, {
 For possible `every` parameter values please refer to
 [`refreshKey`][ref-cube-refreshkey] documentation.
 
+You can also use `every` with `sql`:
+
+<!-- prettier-ignore-start -->
+[[warning | ]]
+| When using `every` and `sql` together, the value of `every` should match
```

Comment:
Is it coming from the core team?

Reply:
That's what I understood from the task in Notion, will confirm with @ovr